### DISCUSSION TITLE
Data points retrieved from Libre history

### DISCUSSION THREAD

**Discussion by Hexonium** [NightscoutFoundation/xDrip#3296] [Ideas]
When the phone loses bluetooth connection to a libre sensor and later reconnects, the missed data points that are retrieves from the Libre history could be displayed using a different pattern. This would be helpful seeing when the connection was lost (especially during the night).

**Comment by Navid200** [comment]
Would you please have a look at the data table?
Is it not already identified there?

https://navid200.github.io/xDrip/docs/Datatables.html

**Comment by Hexonium** [comment]
I don't know if the BG data table contains this info, but I'm suggesting that it could be shown on the main graph (instead of having to check the full events log for instance).

**Reply by Navid200** [reply]
Wait a minute.  We already have that feature.  Just a sec.

**Reply by Navid200** [reply]
Please try this:  https://navid200.github.io/xDrip/docs/Display/ShowBackfilled.html

**Reply by Hexonium** [reply]
That's exactly what I had in mind, thank you! I will give it a try.
